tests/unit/Helpers/Routing.spec.js

```diff
@@ -238,3 +239,53 @@ describe('Open club page', () => {
         });
     });
 });
+describe('Routing::openHome', () => {
+    let router = {
+        currentRoute: {
+            params: {},
+            fullPath: ''
+        }
+    };
+
+    it('Url does not go to iframe', () => {
+        router.currentRoute.params = {
+            match: 'ww',
+            year: '2920',
+            iframe: false
+        };
+        expect(openHome(router.currentRoute.params)).toBe('/');
+    });
+
+    it('Url does not go to iframe, but does go to homepage', () => {
+        router.currentRoute.params = {
+            match: 'ww',
+            year: '2920',
+            iframe: false
+        };
+        expect(openHome(router.currentRoute.params)).not.toBe('/asd');
+    });
+
+    it('Url stays on iframe', () => {
+        router.currentRoute.params = {
+            match: 'ww',
+            year: '2920',
+            iframe: true
+        };
+        expect(openHome(router.currentRoute.params)).toBe('/iframe/ww/2920/');
+        expect(openHome(router.currentRoute.params)).not.toBe('/ww/2920/');
+    });
+
+    it('Url goes to iframe, but of same match and same year', () => {
+        router.currentRoute.params = {
+            match: 'ww',
+            year: '2920',
+            iframe: false
+        };
+        expect(openHome(router.currentRoute.params)).not.toBe(
+            '/iframe/ww/2921/'
+        );
+        expect(openHome(router.currentRoute.params)).not.toBe(
+            '/iframe/fiets/2920/'
+        );
+    });
```

Comment:
Wat test deze test? En mag ook weg?

Reply:
Ja, hier is de vraag, als je naar de ww van 2017 kijkt, en dan op home klikt. wil je dan ww 2017 zien of de nieuwste versie? Ik zou zelf zeggen 2017

Reply:
Maar in principe wordt dat al getest met die andere test waar je checked dat je naar de iframe pagina gaat met jaar 2920

Reply:
Juist kijken naar de if statements lijkt me